.buildkite/pipeline.yml

```diff
@@ -9,6 +9,56 @@ env:
   # work well with podman.
   TESTCONTAINERS_RYUK_DISABLED: "true"
 steps:
+  - key: chart-lint
```

Comment:
Should we just make a standalone lint task that runs `task lint` across everything?